x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -173,3 +173,946 @@ host:keyword | semantic_text_field:semantic_text
 "host2"      | all we have to decide is what to do with the time that is given to us
 "host3"      | be excellent to each other
 ;
+
+case
```

Comment:
tests for [Conditional functions and expressions](https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#esql-conditional-functions-and-expressions)